src/xgov-dapp/src/features/vote/index.tsx

```diff
@@ -440,13 +441,12 @@ function Vote({ sort: sortProp = 'none' }: { sort?: 'ascending' | 'descending' |
                 <Skeleton className="h-40" variant="rectangular" />
               </div>
             )}
-            {votingRoundMetadata?.questions
-              .shuffle()
+            {randomQuestions
               .filter(filterQuestions)
               .sort(sortQuestions)
               .sort(pinPassedQuestions)
-              .map((question, index) => (
-                <div key={index} className="col-span-3 grid grid-cols-1 lg:grid-cols-3 gap-4 bg-white rounded-lg">
+              .map((question) => (
+                <div key={question.id} className="col-span-3 grid grid-cols-1 lg:grid-cols-3 gap-4 bg-white rounded-lg">
```

Comment:
why did we change it to `key={question.id}`?

Reply:
`key` binds state to components in an array. Since the `question` can now move to a different index, we bind state to the unique id of the question